Server/prisma/seed.ts

```diff
@@ -0,0 +1,61 @@
+import { PrismaClient } from "@prisma/client"
+import dotenv from "dotenv"
+import CryptoService from "../src/services/crypto.service"
+import database from "../src/services/database.service"
+import { logger } from "../src/utils/logger"
+
+const prisma = new PrismaClient()
+
+/**
+ * This script initialises the database with first entries.
+ * (See docs: https://www.prisma.io/docs/guides/migrate/seed-database)
+ *
+ * The database schema is generated via `prisma db push`.
+ * This script only initialises the database data if there is no data!
+ */
+async function main() {
+	// Import variables from .env into process.env
+	dotenv.config()
+	const config = process.env
+
+	// If no user exists: Create Initial User
+	// Intial Users credentials are stored in environment variables
+	// for security reasons
+	if ((await database.users.getAll()).length == 0) {
+		const pwd = await CryptoService.produceHash(config.INIT_PWD!)
```

Comment:
I think it is a bad idea to simply assert that an environment variable exists. Even though these are currently defined in `.env`

Reply:
Yeah, maybe checking if the environment variables exists and if not log that initialization couldn't proceed on that part is smarter.

Will change that.
Thanks for the feedback !